Migration test (#485)
nitnelave/lldap: docs/database_migration.md

sed -i -r -e 's/^INSERT INTO "?([a-zA-Z0-9_]+)"?/INSERT INTO `\1`/' \
-e '1s/^/START TRANSACTION;\n/' \

> -e '$aCOMMIT;' /path/to/dump.sql
> 
> sed  -i '1 i\SET FOREIGN_KEY_CHECKS = 0;' ./dump.sql

Reviewer: ```suggestion
-e '$aCOMMIT;' \
-e '1 i\SET FOREIGN_KEY_CHECKS = 0;' /path/to/dump.sql
```

-e 's/^INSERT INTO "?([a-zA-Z0-9_]+)"?/INSERT INTO `\1`/' \
-e '1s/^/START TRANSACTION;\n/' \

> -e '$aCOMMIT;' /path/to/dump.sql
> 
> sed  -i '1 i\SET FOREIGN_KEY_CHECKS = 0;' ./dump.sql
> ```

Reviewer: ```suggestion
-e '$aCOMMIT;' \
-e '1 i\SET FOREIGN_KEY_CHECKS = 0;' /path/to/dump.sql
```
```

Modify your `database_url` in `lldap_config.toml` (or `LLDAP_DATABASE_URL` in the env)
to point to your new database (the same value used when generating schema). Restart
LLDAP and check the logs to ensure there were no errors.

### More details/examples can be seen in the CI process [here](https://raw.githubusercontent.com/nitnelave/lldap/main/.github/workflows/docker-build-static.yml)

> * Postgres L308-327

Reviewer 2: I'm wary of putting line numbers, since any diff in the workflow will lead to outdated line numbers. Can you put maybe job names?

Modify your `database_url` in `lldap_config.toml` (or `LLDAP_DATABASE_URL` in the env)
to point to your new database (the same value used when generating schema). Restart
LLDAP and check the logs to ensure there were no errors.

> ### More details can look into CI proses [here](https://raw.githubusercontent.com/nitnelave/lldap/main/.github/workflows/docker-build-static.yml)

Reviewer: ```suggestion
### More details/examples can be seen in the CI process [here](/.github/workflows/docker-build-static.yml)
```
